clusterloader2/pkg/measurement/common/hubble/manifests/hubble-cli.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hubble-cli-cl2
+  namespace: kube-system
+spec:
+  containers:
+    - name: hubble-cli
+      image: quay.io/cilium/hubble:latest
```

Comment:
is this the image we want to test?

Reply:
I'm not familiar enough with this framework, or is this image replaced later on the `setup` step?